### ISSUE TITLE
Help with install of spot-us

### ISSUE THREAD

**Issue by TimRRingwood** [spot-us/spot-us#2]
I new to 3rd party Ruby code.

I have tried to install open-us for a client and I get errors, it would appears I need install Cucumber first.

I have gotten to the point where trying to run db:migrate I get the message:

```
undefined method `each' for #<Mysql:0x3d99560>
```

What version of mysql and Ruby on Rails is spot-us using?

thanks

**Comment by TimRRingwood** [comment]
I see that libmysql.dl that comes with MySQL 5 doesn't work with Ruby and you need to install an older version

**Comment by eriksundelof** [comment]
Yeah this sounds more like an issue with the mysql gem than the spotus code.
